packages/table/src/fixtures.ts

```diff
@@ -0,0 +1,19 @@
+export const defaultData = [
+  { name: 'Alice', age: 19, color: 'white', location: 'bedford' },
```

Comment:
Let's add some nested objects here for a richer example set.

Reply:
👍